source/fundamentals/crud/read-operations/sort.txt

```diff
@@ -4,135 +4,382 @@ Sort Results
-        .sort(orderBy(ascending("length"), descending("title")))
-        .iterator();
+   import static com.mongodb.client.model.Sorts.ascending;
+
+   // <MongoCollection setup code here>
+   
+   collection.find().sort(ascending("letter"));
+
+Since multiple documents contain the value "a" for the field on which we perform the sort,
+the following documents could be returned in any order:
+
+.. code-block:: json
 
-In the returned documents, notice how the two books with the same 
-length - "Infinite Jest" and "A Dance with Dragons" -
-appear in the reverse of the order shown in the previous example:
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+
+If you need to guarantee a specific sort order for documents that have fields
+with identical values, you can specify additional fields to sort on in the event
+of a tie.
+
+We can specify an ascending sort on the ``letter`` field followed by the
+``_id`` field as follows:
 
 .. code-block:: java
 
-    Document{{_id=1, title=The Brothers Karamazov, length=824, author=Dostoyevksy}}
-    Document{{_id=5, title=Cryptonomicon, length=918, author=Stephenson}}
-    Document{{_id=3, title=Atlas Shrugged, length=1088, author=Rand}}
-    Document{{_id=4, title=Infinite Jest, length=1104, author=Wallace}}
-    Document{{_id=6, title=A Dance with Dragons, length=1104, author=Martin}}
-    Document{{_id=2, title=Les Misérables, length=1462, author=Hugo}}
+   import static com.mongodb.client.model.Sorts.ascending;
 
-For additional information on the classes and methods mentioned on this
-page, see the following API documentation:
+   // <MongoCollection setup code here>
+   
+   collection.find().sort(ascending("letter", "_id"));
 
-- :java-sync-api:`FindIterable <com/mongodb/client/FindIterable.html>`
-- :java-sync-api:`MongoCursor <com/mongodb/client/MongoCursor.html>`
-- :java-docs:`Sorts </apidocs/mongodb-driver-core/com/mongodb/client/model/Sorts.html>`
+The code snippet above returns the documents in the
+:ref:`sample collection <sorts-crud-sort-sample>`  
+in the following order: 
+
+.. code-block:: json
+
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+
+Combining Sort Criteria
+~~~~~~~~~~~~~~~~~~~~~~~
+
+To combine sort criteria, use the ``Sorts.orderBy()`` static factory
+method. This method constructs an object containing an ordered list of sort
+criteria. When performing the sort, if the leftmost sort criteria results in a
+tie, the sort uses the next sort criteria in the list to determine the order.
+
+In the following code snippet, we use the ``orderBy()`` method to combine a
```

Comment:
Since you've described oerderBy() using terms like "ordered list" and "determine the order", the term "combine" used here could be confusing. Maybe you can use different terminology to explain that the sorts you are passing are considered in order?

Reply:
Thanks for the note. I agree, introducing the term "combine" adds unnecessary complexity for learners. Fix: https://github.com/mongodb/docs-java/pull/66/commits/86eef3bcef3c19d3dfd6cdea3fb63f8069624710

Going forward, I will try to be mindful of introducing new terms in an unproductive way, and will try to build empathy for the challenges learners face in being confronted with a new term.